src/quo2/components/graph/wallet_graph/utils.cljs

```diff
@@ -0,0 +1,29 @@
+(ns quo2.components.graph.wallet-graph.utils)
+
+(defn find-highest-value
+  [maps]
+  (reduce (fn [max-value map]
+            (let [value (:value map)]
+              (if (nil? max-value) ; Check if it's the first element
+                value
+                (if (> value max-value)
+                  value
+                  max-value))))
+          nil
```

Comment:
I'm a bit out of context here, but can I bet negative? Can we just go with with `-1` instead of having `if` later?